parameters.cpp

```diff
@@ -467,7 +471,43 @@ bool Parameters::getParameters(){
    Readparameters::get("AMR.vel_refinement_criterion",P::amrVelRefCriterion);
    Readparameters::get("AMR.refine_limit",P::amrRefineLimit);
    Readparameters::get("AMR.coarsen_limit",P::amrCoarsenLimit);
+  
+  /*Read Blur Passes per Refinement Level*/
+   Readparameters::get("AMR.filterpasses",P::blurPassString);
    
+   // Construct Vector of Passes used in grid.cpp
+   bool isEmpty = blurPassString.size()==0;
+   if (!isEmpty){
+
+      for (auto i : blurPassString){
+         P::numPasses.push_back(std::stoi(i));
+      }
+
+      
+      // Reverse Sort and Get the maximum number of filter passes
+      sort(numPasses.begin(), numPasses.end(), greater<int>());
+      P::maxNumPasses = *max_element(P::numPasses.begin(), P::numPasses.end());
```

Comment:
@markusbattarbee It belongs to Parameteres and it is also calculated as the max element from numPasses above.

Reply:
right, well it should be either or. A parameter is an input thing, not something calculated in the code.

Reply:
Fixed!